src/ci.js

```diff
@@ -116,6 +116,14 @@ exports.getVars = function() {
     }
     return result;
   }
+  // Github actions
+  if(env.CI === 'true' && env.GITHUB_ACTIONS === 'true'){
+    return {
+      build: env.GITHUB_RUN_NUMBER,
+      branch: env.GITHUB_REF,
```

Comment:
@luisVelasquezSauce can you also remove `refs/heads/` prefix if it exists? similar to here: https://github.com/screener-io/screener-runner/blob/master/src/ci.js#L105

So the branch that displays in the UI is `main`, and not `refs/heads/main`

Reply:
Updated code and tests